core/src/main/scala/org/http4s/util/Renderable.scala

```diff
@@ -120,3 +123,18 @@ class StringWriter(size: Int = 64) extends Writer {
 
   def result(): String = sb.toString
 }
+/** [[Writer]] that will result in a `ByteVector`
+  * @param bv initial ByteVector`
+  */
+case class ByteVectorWriter(private var bv:ByteVector = ByteVector.empty) extends Writer {
```

Comment:
I've been wondering when we would get a second `Writer` type! This is a very good use case.